.github/workflows/publish.yml

```diff
@@ -4,6 +4,10 @@ permissions: {}
 
 on:
   workflow_dispatch:
+    inputs:
+      ref:
+        type: string
+        required: false
```

Comment:
does this make manual dispatch "just work" without arguments? or should the default value be 'publish'?

Reply:
Yes exactly. If not provided, it pulls the latest version of the branch.